sdk/core/azure-core/src/main/java/com/azure/core/util/Configuration.java

```diff
@@ -83,7 +88,7 @@
     /**
      * Name of the Azure cloud to connect to.
      */
-    public static final String PROPERTY_AZURE_CLOUD = "AZURE_CLOUD";
```

Comment:
Removing/renaming a public env variable that is already released is considered a breaking change. Can this env var be retained and `PROPERTY_AZURE_AUTHORITY_HOST` be added as a new env var?